packages/grid/frontend/src/lib/jsserde/jsClient.svelte

```diff
@@ -53,7 +52,7 @@
     }
 
     get metadata() {
-      return fetch(this.url + '/api/v1/new/metadata_capnp')
+      return fetch('http://localhost:8081/api/v1/syft/metadata')
```

Comment:
Use the API path instead of the invalid localhost url.

Reply:
Fixed!